src/app/features/tech-record/components/tech-record-title/tech-record-title.component.html

```diff
@@ -1,4 +1,24 @@
 <dl *ngIf="currentTechRecord$ | async as currentTechRecord" class="govuk-summary-list">
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">Vehicle type</dt>
+    <dd id="vehicleType" class="govuk-summary-list__value">
+      {{ (editableTechRecord$ | async)?.vehicleType ?? '' | uppercase | defaultNullOrEmpty }}
+    </dd>
+    <dd class="govuk-summary-list__actions"></dd>
+    <dd class="govuk-summary-list__actions">
+      <a
```

Comment:
as discussed, preferably use `<app-button>` for this. it can work as a link or button.

Reply:
Changed to app-button with design="link"